kooritea/fcmfix
# Fixed auto start issue

Updated hook point in AutoStartFix to address the issue that app won't be able to receive notification without auto start enabled in app config. 

Now app can receive notification even if auto start is disabled. 

Test steps
1. Disable auto start in app config
2. Force stop app
3. Send notification from other devices
4. Confirm notification can be received on device w/ Android 14/Hyper OS
